Lib/sqlite3/dbapi2.py

```diff
@@ -84,6 +84,20 @@ def convert_timestamp(val):
 
 register_adapters_and_converters()
 
+# bpo-24464: enable_shared_cache was deprecated in Python 3.10.  It's
+# scheduled for removal in Python 3.12.
+_old_enable_shared_cache = enable_shared_cache
+def enable_shared_cache(enable):
+    import warnings
+    msg = ("""
+        enable_shared_cache is deprecated and will be removed in Python 3.12.
+        Shared cache is strongly discouraged by the SQLite 3 documentation.
+        If shared cache must be used, open the database in URI mode using
+        the cache=shared query parameter.
+    """)
```

Comment:
Cosmetic notes (can be ignored): the parens are not needed around multiline strings; by default the warning will be printed on 4 lines and with extra leading whitespace, which is not always nice to read in logs and other.

Reply:
Removing the indent and keeping it on one line seems to be consistent with other deprecation warnings. Thanks!